rbac/rbac/settings.py

```diff
@@ -346,6 +346,15 @@
 
 ROLE_CREATE_ALLOW_LIST = ENVIRONMENT.get_value("ROLE_CREATE_ALLOW_LIST", default="").split(",")
 
+DEFAULT_DUAL_WRITE_APP_DENY_LIST = [
+    "approval",
+    "cost-management",
+    "playbook-dispatcher",
+]
+DUAL_WRITE_APP_DENY_LIST = ENVIRONMENT.get_value(
```

Comment:
So this will utilize the default list _or_ a list that you define, correct? We do have a pattern already for some other app lists: https://github.com/RedHatInsights/insights-rbac/blob/d8051fd8ff1041f054bacdb657ce135e6b8977cc/deploy/rbac-clowdapp.yml#L733-L734 where instead of hard-coding the default list, we can have a default in the template, parameterized, and then we can set it per environment as well. Thoughts?

Also, is this not being consumed (yet)?

Reply:
Yeah I appreciate that consistency. It was hard coded before so I sort of ported it "as is" but that's not really an excuse!

It's a bit burred in the diff but it's used in sharedSystemRolesReplicatedRoleBindings.py:

```python
def is_for_enabled_app(perm: V1permission):
    """Return true if the permission is for an app that should migrate."""
    return perm.app not in settings.DUAL_WRITE_APP_DENY_LIST
```

I'll make the update!

Reply:
Ahh!! Missed that, thanks! 😄